zipline/finance/risk/cumulative.py

```diff
@@ -281,6 +286,12 @@ def update(self, dt, algorithm_returns, benchmark_returns, leverage):
         self.max_leverage = self.calculate_max_leverage()
         self.max_leverages[dt_loc] = self.max_leverage
 
+        self.cvar[dt_loc] = conditional_value_at_risk(
+            returns=pd.DataFrame(self.algorithm_returns[-CVAR_LOOKBACK_DAYS:]),
```

Comment:
How often is this called? Any concern about the speed of making a `DataFrame` here?

When we originally moved the other metrics to empyrical, they slowed down the zipline test suite dramatically because of all the `Series` constructions, whereas zipline had been using raw numpy arrays previously. Eventually empyrical was updated to support either an `ndarray` or a `Series/DataFrame` as input to each function, without having to wrap the arrays in pandas objects. This dropped the test times back down.